spec/validators/ingested_numericality_validator_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe IngestedNumericalityValidator do
+  let(:sheet_class) do
+    Class.new(Framework::Sheet) do
+      extend ActiveModel::Naming
+
+      def self.name
+        'Validator'
+      end
+
+      field 'test', :string, ingested_numericality: true
+    end
+  end
+
+  it 'validates integers' do
+    [-42, 0, 1234].each do |valid_integer|
+      instance = sheet_class.new_from_params('test' => valid_integer)
+
+      expect(instance).to be_valid
+    end
+  end
+
+  it 'validates floats' do
+    [-1.23, 0.0, 99.99].each do |valid_float|
```

Comment:
Do we want to test that string values of numbers validate here?

Reply:
Anything that appears to be an integer or float gets coerced into a decimal at ingest. We also do a lot of clean up at that point.